src/pages/en/reference/api-reference.mdx

```diff
@@ -830,7 +830,7 @@ A function to compile a given Markdown or MDX document for rendering. This retur
 
 - `<Content />` - A component used to render the document's contents in an Astro file.
 - `headings` - A generated list of headings, [mirroring Astro's `getHeadings()` utility](/en/guides/markdown-content/#exported-properties) on Markdown and MDX imports.
-- `injectedFrontmatter ` - An object of frontmatter [injected via remark or rehype plugins](/en/guides/markdown-content/#example-injecting-frontmatter). Set to type `any`.
+- `injectedFrontmatter ` - An object of frontmatter [modified via remark or rehype plugins](/en/guides/markdown-content/#modifying-frontmatter-programmatically). Set to type `any`.
```

Comment:
Just checking that you still want to name this `injectedFrontmatter` even though all our descriptions now refer to it as "modified frontmatter"?  (I understand that changing the code is probably a bigger deal than changing words in the docs. Just pointing it out!)

Reply:
I'm aware, and I'm debating a good name for it 😓 
`pluginFrontmatter` is the rough name I'm going with?

Reply:
Update: decided on `remarkPluginFrontmatter`! Remark is the umbrella for "remark and rehype plugins" according to the unified spec. Also want to clarify this is the frontmatter from remark _only,_ and won't reflect any schemas you have applied.